### ISSUE TITLE
Homework for next week

### ISSUE THREAD

**Issue by sarakenney** [AsaSmith/bsomethingmore#1]
1. Find inspiration for the home page layout.
2. Find inspiration for your personal about page keeping in mind all pages will have similar layout.
3. Write your bio.

**Comment by asasmith** [comment]
I thought layout can vary.

**Comment by sarakenney** [comment]
For personal pages, maybe but didn't we say we wanted the header or some piece of the site to look consistent on all pages?

**Comment by debihoward** [comment]
What about a small navigation bar at the top so they can easily get
anywhere on the site without hitting the back button?  That will keep
consistency throughout.

_Debi Howard_
P  410-598-3720
P.O. Box 501  |  Severna Park, MD  |  21146

**Comment by asasmith** [comment]
I agree with having consistent elements (nav, footer etc) across all pages, especially if our bio pages have some visual differences in the rest of their layout.

**Comment by asasmith** [comment]
Also, thoughts on setting up blog post templates? We could create a few, and leave the content out, then just copy and paste content into the right places when we're authoring posts.

**Comment by debihoward** [comment]
Haven't thought that far yet.

_Debi Howard_
P  410-598-3720
P.O. Box 501  |  Severna Park, MD  |  21146

**Comment by sarakenney** [comment]
we could create a branch just for the blog post template, then each person could clone it over and make it their own branch for their posts. is that what you're thinking?
